src/utils.js

```diff
@@ -19,6 +18,12 @@ export function isEmailValid(email) {
   return new RegExp(EMAIL_REGEX).test(email);
 }
 
+export function isPasswordValid(password) {
+  const isEightLettersOrMore = password.length >= 8;
+  const hasNumberAndCharacter = password.match(/[0-9]/g) && password.match(/[a-zA-Z]/gi);
+  return isEightLettersOrMore && hasNumberAndCharacter;
+}
+
 export function togglePassword(input, toggleButton) {
   if (input.getAttribute("type") === "password") {
     input.setAttribute("type", "text");
```

Comment:
조건문의 내부와 바깥이 유사한 코드로 보여집니다.
아래와 같이 바꿔보시는건 어떨까요?

```suggestion
const isPassword = input.getAttribute("type") === "password";
const eyeIconUrl = `./images/eye-${isPassword ? "on" : "off"}.svg`;
input.setAttribute("type", isPassword ? "text" : "password");
toggleButton.querySelector("img").setAttribute("src", eyeIconUrl);
```